userspace/falco/CMakeLists.txt

```diff
@@ -84,9 +84,7 @@ set(
   "${YAMLCPP_LIB}"
 )
 
-if(USE_BUNDLED_DEPS)
-  list(APPEND FALCO_DEPENDENCIES yamlcpp)
-endif()
+list(APPEND FALCO_DEPENDENCIES yamlcpp)
```

Comment:
Can you elaborate more on why `if(USE_BUNDLED_DEPS)` is not required anymore? :thinking: